source/common/http/http2/codec_impl.h

```diff
@@ -585,7 +585,9 @@ class ConnectionImpl : public virtual Connection,
     // written out before force resetting the stream, assuming there is enough H2 connection flow
     // control window is available.
     bool useDeferredReset() const override { return true; }
-    StreamDecoder& decoder() override { return *request_decoder_; }
+    StreamDecoder* decoder() override {
+      return request_decoder_handle_->get().ptr();
```

Comment:
get() should get you the raw pointer.   Is the .ptr() redundant?

Reply:
`get()` gets an `OptRef`: https://github.com/envoyproxy/envoy/blob/1e68b78b9f26e4d93b01faae257158b3ec54798a/source/common/http/conn_manager_impl.h#L533, on which you have to call `ptr()` to get the raw pointer.